eng/Generate.ps1

```diff
@@ -238,6 +238,22 @@ if (!($Exclude -contains "Samples"))
     }
 }
 
+# Sample for cadl project
+$cadlSampleProjectName = 
+    'AnomalyDetector'
+
+if (!($Exclude -contains "Samples"))
+{
+    foreach ($projectName in $cadlSampleProjectName)
+    {
+        $projectDirectory = Join-Path $repoRoot 'samples' $projectName
+        $cadlMain = Join-Path $projectDirectory "main.cadl"
+        $cadlClient = Join-Path $projectDirectory "main.cadl"
```

Comment:
What is the difference between `cadlMain` and `cadlClient`? They look the same here.

Reply:
cadlClient refers to client.cadl. updated